config/install/file.settings.yml

```
@@ -0,0 +1,15 @@
+_core:
+  default_config_hash: 2bwjZB1IjEYbppgZT3g7YW_5h_LDZYNa3DaDEfWX82U
```

These lines can be deleted:
```suggestion
_core:
  default_config_hash: 2bwjZB1IjEYbppgZT3g7YW_5h_LDZYNa3DaDEfWX82U
```

---

Thanks Brian, will update.